addons/l10n_it_edi_withholding/data/account.tax.group.csv

```diff
@@ -0,0 +1,4 @@
+id,name,sequence,preceding_subtotal
+tax_group_pension_fund,Pension Fund,0
```

Comment:
If we update the tax group (maybe in XML) before withholding, we could make that field translate True and put those in an XML...  (that way it will be in the pot of l10n_it_edi_withholding)